dml/src/lib.rs

```diff
@@ -258,6 +258,11 @@ impl DmlWrite {
         self.tables.iter().map(|(k, v)| (k.as_str(), v))
     }
 
+    /// Moves the table names and mutable batches out of the map
+    pub fn tables_into_iter(self) -> impl Iterator<Item = (String, MutableBatch)> {
```

Comment:
```suggestion
    pub fn into_tables(self) -> impl Iterator<Item = (String, MutableBatch)> {
```

The "Rusty" naming is `into_x` for transforming a container into an iterator of it's parts. See `{HashMap,BTreeMap}::{into_keys,into_values}` for example.